locust/test/test_runners.py

```diff
@@ -717,7 +717,7 @@ def addWorker():
             def checkRebalancedTrue():
```

Comment:
In Python, `snake_case` is preferred. There's some old library such as `unittest` that uses `camelCase` (i.e. `self.assertTrue`), but this is only because the standard library needs to keep backward compatibility.

Reply:
Ok, this code was already like that. Sorry. Nevermind then